raiden/network/proxies/token.py

```diff
@@ -211,19 +212,19 @@ def transfer(self, to_address: Address, amount: TokenAmount) -> None:
 
             if not is_succesful:
                 if not was_transaction_successfully_mined(transaction_mined):
-                    failed_at_number = transaction_mined.receipt["blockNumber"]
+                    failed_at_block_number = BlockNumber(transaction_mined.receipt["blockNumber"])
                 else:
-                    failed_at_number = self.client.get_block(BLOCK_ID_LATEST)["blockNumber"]
+                    failed_at_block_number = self.client.get_block(BLOCK_ID_LATEST)["blockNumber"]
```

Comment:
Shouldn't this also be wrapped in `BlockNumber()`?

Reply:
Good point, according to `BlockData`, there is no key `blockNumber` in a block info, just `number`. That's why I didn't fix it yet, but it's in the list of mypy errors to fix.